docs/source-app/core_api/lightning_work/index.rst

```diff
@@ -2,7 +2,8 @@
 Lightning Work
 ##############
 
-**Audience:** Users who want to know how Lightning Work works under the hood 🤯.
+| **Audience:** Users who want to know how Lightning Work works under the hood 🤯.
+| **Limitation:** At this time there is a 20GB disk size limit for Lightning Works.
```

Comment:
There must be a typo. 20 GB would be too small.

Reply:
@hhsecond can you confirm or deny this?